example.js

```
@@ -5,4 +5,16 @@ const captureWebsite = require('.');
 
 (async () => {
 	await captureWebsite.file('https://sindresorhus.com', 'example.png');
+
+	await captureWebsite.file('https://trello.com', 'example_iphone_x.png', {
+		emulateDevice: 'iPhone X'
+	});
+	await captureWebsite.file('https://trello.com', 'example_iphone_x_with_frame.png', {
+		emulateDevice: 'iPhone X',
+		deviceFrame: true
+	});
+	await captureWebsite.file('https://trello.com', 'example_iphone_6_with_frame.png', {
+		emulateDevice: 'iPhone 6',
```

This would not work as there is no iPhone 6 frame, so why are you showing it?

---

since there is no frame image for iphone6, It will be equivalent to `deviceFrame: false` !

---

It shouldn't silently ignore what the user is requesting: https://github.com/sindresorhus/capture-website/pull/27#discussion_r287723192

---

@sindresorhus - Should I add `console.log or console.error` to present error to user?

---

No, reusable modules should never use console.log directly. It should throw an error.

---

I have added `throw new Error` on incompatible options.